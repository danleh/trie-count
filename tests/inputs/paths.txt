./Cargo.toml
./Cargo.lock
./src/main.rs
./src/options.rs
