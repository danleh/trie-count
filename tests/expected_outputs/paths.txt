4 ./
  2 src/
    1 main.rs
    1 options.rs
  1 Cargo.toml
  1 Cargo.lock